### PR TITLE
Pisi GNU/Linux is added

### PR DESCRIPTION
**Pull request by ayhanyalcinsoy**

### CODE REVIEW

File: data/distro.yml [hunk 1]
```diff
@@ -605,3 +605,26 @@
...
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
```
Comment (AsciiWolf) [inline data/distro.yml:626]: Is graphical installation of Flatpak apps possible on this distro? If not, please mention it here (see setup pages of other distros for example).

File: data/distro.yml [hunk 2]
```diff
@@ -605,3 +605,27 @@
...
+        <p>Note: For now, graphical installation of Flatpak apps are not be possible on Pisi GNU/Linux.</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: Thanks. Small typo: **are** not **be** possible (Should be: is not possible)

File: data/distro.yml [hunk 3]
```diff
@@ -605,3 +605,27 @@
...
 
+- name: "Pisi GNU/Linux"
+  logo: "pisi.svg"
+  info:  >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>A flatpak package is available in Pisi 2.1 and newer. To install it, run the following as root:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo pisi it flatpak
```
Comment (AsciiWolf) [inline data/distro.yml]: Please, change the prompt character from `#` to `$`. There is no point in using sudo on a root prompt.

File: data/distro.yml [hunk 4]
```diff
@@ -605,3 +605,27 @@
...
+        </code></pre>
+      </li>
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flathub is the best place to get Flatpak apps. To enable it, run:</p>
+        <pre><code>
+          <span class="unselectable">#</span> sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```
Comment (AsciiWolf) [inline data/distro.yml]: Same as the command above + please remove `sudo`, it should not be needed and is not recommended for Flatpak commands (Polkit is used to elevate their rights).

File: data/distro.yml (hunk 2 above)
Comment (AsciiWolf) [reply data/distro.yml]: One more smaller thing: please change the `For` to a lowercase (`for`).

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml (hunk 4 above)
Comment (ayhanyalcinsoy) [reply data/distro.yml]: On Pisi GNU/Linux users need to superuser rights to writing to main file system. Therefore they must to use "sudo" command before installating/removing an app.

File: data/distro.yml [hunk 5]
```diff
@@ -605,3 +605,27 @@
...
+        </code></pre>
+      </li>
+      <li>
+        <h2>Restart</h2>
+        <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+        <p>Note: For now, graphical installation of Flatpak apps is not possible on Pisi GNU/Linux.</p>
```
Comment (AsciiWolf) [inline data/distro.yml]: One more smaller thing: please change the `For` to a lowercase (`for`).

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/distro.yml (hunk 4 above)
Comment (AsciiWolf) [reply data/distro.yml]: So Polkit is not supported there?

Comment (AsciiWolf) [reply data/distro.yml]: Let me know what is the state of Polkit support on your distro. If elevating rights using Polkit it really is not supported, I think that using sudo for the flatpak command makes sense. I just hope it won't cause any other issues.

Comment (ayhanyalcinsoy) [reply data/distro.yml]: In fact, we also use Polkit. But Pisi GNU/Linux uses own base system and on terminal it request sudo command.

**Comment by AsciiWolf** [comment]
One more small change: Please remove empty lines [here](https://github.com/flatpak/flatpak.github.io/blob/0c5a25e2f5a1b69cbfbe0bd320e3f830840893b3/data/distro.yml#L607) and [here](https://github.com/flatpak/flatpak.github.io/blob/0c5a25e2f5a1b69cbfbe0bd320e3f830840893b3/data/distro.yml#L574). Otherwise LGTM!

**Review by AsciiWolf — CHANGES_REQUESTED** [review]
One more small change: Please remove empty lines [here](https://github.com/flatpak/flatpak.github.io/blob/0c5a25e2f5a1b69cbfbe0bd320e3f830840893b3/data/distro.yml#L607) and [here](https://github.com/flatpak/flatpak.github.io/blob/0c5a25e2f5a1b69cbfbe0bd320e3f830840893b3/data/distro.yml#L574). There should be only one empty line. Otherwise LGTM! I will merge the PR after this last change.

**Comment by AsciiWolf** [comment]
Thanks!

**Pull request merged**